actions/templates/install/index.js

```diff
@@ -149,12 +150,12 @@ async function main (params) {
     const credentialFlowType = credentials[0].flowType;
     let createIntegrationResponse = {};
 
-    if (credentialFlowType === 'adobeid') {
+    if (credentialFlowType.toLowerCase() === CREDENTIAL_FLOW_TYPE_ADOBEID) {
       // form integration request body
       const createAdobeIdIntegrationReqBody = {
         name: String(body.projectName),
         description: String(body.description ? body.description : `Created from template ${template.name}`),
-        platform: String(mapAdobeIdCredentialTypeToPlatformType[credentialType]),
+        platform: String(mapAdobeIdCredentialTypeToPlatformType[credentialType.toLowerCase()]),
```

Comment:
nit: Can this be null? It appears you cover the case where it is not a string.  Either it is always defined, and always a string, or not.